cp-ksqldb-server/Dockerfile.ubi8

```diff
@@ -36,10 +40,33 @@ COPY --chown=appuser:appuser include/etc/confluent/docker/* /etc/confluent/docke
 RUN mkdir -p /etc/${COMPONENT} /etc/${COMPONENT}/secrets /usr/logs \
   && chown appuser:appuser -R /etc/${COMPONENT} /usr/logs
 
-RUN echo "===> Installing confluent-hub..." \
-  && wget http://client.hub.confluent.io/confluent-hub-client-latest.tar.gz \
-  && tar xf confluent-hub-client-latest.tar.gz \
-  && rm confluent-hub-client-latest.tar.gz
+RUN yum -q -y update \
```

Comment:
This bit seems to be similar to the same bit from the other file - is it possible to put it in a script and call the script from both?

Reply:
it is similar, i will look at running a script to update and add the repo (this code is duplicated across our images generally though, so might tackle as a follow-on JIRA to do this consistently vs. just fixing in this repo)